test-framework-docker/executor/src/main/java/com/containersolutions/mesoshelloworld/executor/FrameworkTask.java

```diff
@@ -0,0 +1,71 @@
+package com.containersolutions.mesoshelloworld.executor;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import org.apache.mesos.ExecutorDriver;
+import org.apache.mesos.Protos;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.InetSocketAddress;
+
+import static com.containersolutions.mesoshelloworld.executor.FrameworkExecutor.RESPONSE_STRING;
+
+public class FrameworkTask extends Thread {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FrameworkTask.class);
+
+    private final Protos.TaskInfo task;
+    private final ExecutorDriver driver;
+
+    public FrameworkTask(ExecutorDriver driver, Protos.TaskInfo task) {
+        this.driver = driver;
+        this.task = task;
+    }
+
+    public void run() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1161)